_includes/nav.html

```diff
@@ -1,4 +1,13 @@
 <header class="header">
+  {% if page.url == '/' or page.url == '/program/' %}
+  <div class="announcement relative bg-yellow color-black" role="alert">
+    <div class="container">
+      <p>
+        <a href="/livestream/" class="fw-bold">Watch the 2019 conference live stream</a>
```

Comment:
Can we add this somewhere?

```
Also available at <a href="https://live.mesh.world">live.mesh.world</a> and <a href="https://livestream.com/internetsociety/ournetworks">Internet Society</a>
```

Reply:
See my comment in the meshtv chat -- I think we should tweet about the restreams versus adding them to the announcement.